src/client/components/Avatar/index.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+
+import React from 'react'
+import { default as MaterialAvatar } from 'material-ui/Avatar'
+
+export type Props = {
+  style?: Object,
+  avatar: string,
+};
+
+const avatarStyle = {
+  borderRadius: '20%',
+  float: 'left',
+  display: 'table-column',
+  height: '40px',
+  width: '40px',
+  color: 'rgba(157, 156, 156, 0.58)',
+}
```

Comment:
Regarding CSS topic, I guess lot's of you confused. So no work related CSS modularity done in the project yet, for the reason that styling and design haven't been finalized yet and maintaining css files adds on component development time. 

Loading `.css` file for the react component doesn't guarantee you that another react component accidentally will not override your styles. To get this right, we will introduce CSS plugins e.g. postCss, or will continue using inline styles.

There is no performance impact on using inline styles either. So if you sure there is no style which can override your one then move styles to css files in advance, if you are afraid of issues, leave them inline.